upi/openstack/network.yaml

```diff
@@ -38,10 +38,11 @@
       allocation_pool_end: "{{ os_subnet_range | ipaddr('last_usable') }}"
     when: os_networking_type != "CiscoACI"
 
-  - name: 'Create the cluster address-scope'
-    command:
-      cmd: "neutron address-scope-create node_network_address_scope 4 --apic:distinguished_names type=dict VRF={{ aci_cni['network_interfaces']['node']['vrf'] }}"
-    when: os_networking_type == "CiscoACI"
+    #  - name: 'Create the cluster address-scope'
+    #    command:
+    #      cmd: "neutron address-scope-create node_network_address_scope 4 --apic:distinguished_names type=dict VRF={{ aci_cni['network_interfaces']['node']['vrf'] }}"
+    #    when: os_networking_type == "CiscoACI"
+    #    ignore_errors: True
```

Comment:
I think you disabled this for testing, but should be uncommented. ;-)

Reply:
ahh right, nice catch, thanks, resolving it